ondewo/nlu/aiservices.proto

```diff
@@ -281,3 +286,68 @@ message XLNetAugEnrichmentConfig {
 
     int32 execution_order = 3;
 }
+
+// The request for intent classification.
+message ClassifyIntentsRequest {
+    // the parent of the request
+    // Format: `projects/<Project ID>`.
+    string parent = 1;
+    // the input text
+    string text = 2;
+    // the input language
+    string language_code = 3;
+    // Optional: to use the contexts with names listed in the field `context_names` during the classification
+    bool active_contexts = 4;
+    // Optional: names of the contexts for the intent classification
+    repeated string context_names = 5;
+    // Optional: Which mode to use:
+    // EXCLUSIVE - skip algorithms listed in `algorithms` field,
+    // INCLUSIVE - run ONLY algorithms listed in `algorithms` field,
+    // UNSPECIFIED - default mode, described in intent_ranking_match.py
+    Mode mode = 6;
+    // Optional: Algorithm list
+    repeated IntentAlgorithms algorithms = 7;
+}
+
+enum Mode {
+    UNSPECIFIED = 0;
+    EXCLUSIVE = 1;
+    INCLUSIVE = 2;
+}
+
+enum IntentAlgorithms {
+    OndewoIntentExactContextDetector = 0;
+    OndewoIntentExactMatch = 1;
+    OndewoIntentNamedExactMatch = 2;
+    OndewoIntentSimilarityMatch = 3;
+    OndewoIntentNamedSimilarityMatch = 4;
+    OndewoIntentLogisticRegression = 5;
+    OndewoIntentSupportVectorClassifier = 6;
+    OndewoIntentRandomForestClassifier = 7;
+    OndewoIntentFastTextClassifier = 8;
+    OndewoIntentMachineLearningMatch = 9;
+    OndewoIntentBertClassifier = 10;
+    OndewoIntentMetaClassifier = 11;
+    OndewoIntentSnipsClassifier = 12;
+    IntentExitDetector = 13;
```

Comment:
Are we looking to exclude default classifiers (that are actually only annotation) like `DefaultFallback` (only trigger if nothing else detected an intent), `IntentParameterMatch`, `DeviationEnding` and `ExitDetector`